### PR TITLE
New integration: Ezviz camera

### PR DESCRIPTION
**Pull request by BaQs**
**Description:**
Ezviz camera platform

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#30378

## Checklist:

- [x ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/ezviz.markdown [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: "Ezviz Camera"
+description: "Integrate Ezviz camera within Home Assistant."
+logo: ezviz.png
+ha_release: 0.103.0
```
Comment (klaasnicolaas) [inline source/_integrations/ezviz.markdown]:
```suggestion
ha_release: 0.104
```

File: source/_integrations/ezviz.markdown [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+The configuration (see below) needs your Ezviz username & password you use within the Ezviz mobile application.
+
+Next, add the Ezviz camera to your `configuration.yaml` file like below:
+
+```yaml
+# Example configuration.yaml entry
+camera:
+  - platform: ezviz
+    username: 'user@mail.com'
```
Comment (klaasnicolaas) [inline source/_integrations/ezviz.markdown]:
```suggestion
    username: USERNAME
```

File: source/_integrations/ezviz.markdown [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    password: 'your_password_here'
```
Comment (klaasnicolaas) [inline source/_integrations/ezviz.markdown]:
```suggestion
    password: PASSWORD
```

File: source/_integrations/ezviz.markdown [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    cameras:
+      D12345678:
+        username: admin
```
Comment (klaasnicolaas) [inline source/_integrations/ezviz.markdown]:
```suggestion
        username: CAMERA_USERNAME
```

File: source/_integrations/ezviz.markdown [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        password: ABCDEF
```
Comment (klaasnicolaas) [inline source/_integrations/ezviz.markdown]:
```suggestion
        password: CAMERA_PASSWORD
```

File: source/_integrations/ezviz.markdown [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+      D87654321:
```
Comment (klaasnicolaas) [inline source/_integrations/ezviz.markdown]: Maybe that one cam example is more then enough?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/ezviz.markdown [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+      type: string
+    password:
+      description: The Ezviz verification code.
+      required: true
+      type: string
+{% endconfiguration %}
+
+## Lovelace
+
+If you want a shiny lovelace card, with proper buttons, here is a snippet of a lovelace card:
```
Comment (frenck) [inline source/_integrations/ezviz.markdown]:
```suggestion
If you want a shiny Lovelace card, with proper buttons, here is a snippet of a Lovelace card:
```

**Review by frenck — APPROVED** [review]
There we go! Thanks, @BaQs! 👍

**Pull request merged**